LiskHQ/lisk-hub
# Add vote launch protocol info box

### What was the problem?
We needed an info box for the selected votes like in nano

### How did I fix it?
I used the existing component `src/components/voteUrlProcessor/voteUrlProcessor.js` that now returns the info box.
Since we didn't receive the icons yet, I couldnt add them.

### How to test it?
try e.g. 
`/delegates?unvotes=genesis_2,genesis_88,genesis_51&votes=genesis_72,genesis_60,genesis_77?fdgds,erty65trg`

Check on mobile as well, go through the whole voting process. Make sure it looks fine with many votes as well.

### Review checklist
- The PR solves #742
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
